index.html

```diff
@@ -691,6 +691,20 @@ <h3>Options</h3>
 Likewise, an instance configuration MAY require that clients include some `options`
 properties.
       </p>
+      <h4>Options Extensibility</h4>
+      <p>
+Implementations MAY extend an `options` object with additional properties.
+Property keys can be any string but they should be appropriately namespaced (for example by using URIs)
+in order to avoid collisions with extensions in other implementations.
```

Comment:
I don't know if we need to provide this advice or if it should be adjusted somewhat. We might want to discuss this bit on the next call.

It seems the danger here would actually be with a conflict with something that gets standardized, not with different implementations. This is because if a different implementation were to be swapped in, you'd certainly first stop using any previous-implementation-specific options, regardless of their names. You'd then add any new-implementation-specific options (and it then wouldn't matter whether the same names were used or not). In no case would you be using both implementations at once (or not know which one you were talking to).

Reply:
Until @dlongley's concern above is addressed, this text should be refined a bit, a la --
```suggestion
Property keys can be any string, but they should be appropriately namespaced (by using URIs, for example)
to avoid collisions with other extensions.
```

Reply:
Yeah, I hesitate at saying that the extensibility mechanism is to use full URIs as well. Feels like just text strings could work fine here, and perhaps a registry? (though I shudder at the need for a registry)

Reply:
(also, I left a `should` in there, but if it remains, it should be change to `SHOULD` or `ought to`)

Reply:
```suggestion
Property keys can be any string, but they SHOULD be appropriately namespaced (by using URIs, for example)
to avoid collisions with other extensions.
```
```suggestion
Property keys can be any string but they should be appropriately namespaced (for example by using URIs)
in order to avoid collisions with extensions in other implementations.
```

Reply:
or
```suggestion
Property keys can be any string but they ought to be appropriately namespaced (for example by using URIs)
in order to avoid collisions with extensions in other implementations.
```